bin/index.js

```diff
@@ -26,9 +26,19 @@ const startHTTPServer = require("../src/httpServer.js");
 const i18n = require("../src/i18n");
 const { getRegistryURL, loadNsecureCache, writeNsecureCache, formatBytes } = require("../src/utils");
 const { depWalker } = require("../src/depWalker");
-const { hydrateDB, deleteDB } = require("../src/vulnerabilities");
+const { VulnerabilityStrategy } = require("../src/vulnerabilities/vulnerabilitiesSource");
+const { VULN_MODE_NPM_AUDIT, VULN_MODE_DB_SECURITY_WG } = require("../src/vulnerabilities/strategies");
+const NPMAuditStrategy = require("../src/vulnerabilities/strategies/npm-audit.js");
 const { cwd, verify } = require("../index");
 
+/**
+ * default Strategy is hard initialized, but can be changed at runtime
+ * default Strategy is {@function SecurityWGStrategy}
+ */
+let vulnStrategy = VulnerabilityStrategy();
```

Comment:
We should avoid declaring in global and instead use getVulnerabilityStrategy when we need the strategy?